libsolidity/interface/StandardCompiler.h

```diff
@@ -89,6 +88,10 @@ class StandardCompiler
 		bool viaIR = false;
 	};
 
+	/// Configures @p _compilerStack according to the input settings in @p _inputsAndSettings.
+	static void configure(InputsAndSettings const& _inputsAndSettings, CompilerStack& _compilerStack);
```

Comment:
@cameel this is a bit tricky. I generally agree that configuration is best placed at object construction, but in here in case, I simply moved the function body of that configure() function out of the `compileSolidity` function, which is invoked by `compile()` which may or may NOT contain settings.

I've moved this configuration step out because in the LSP PR we also need it (without code duplication). We could do that in a second step (different PR only taking care of further improving the CompilerStack API) what do you think?